nixos/modules/misc/ids.nix

```diff
@@ -346,6 +346,7 @@ in
       zoneminder = 314;
       paperless = 315;
       #mailman = 316;  # removed 2019-08-30
+      icecc = 317;
```

Comment:
You shouldn't assign static id for a user unless needed. Use of `isSystemUser` is preferred.